catt/controllers.py

```diff
@@ -22,29 +23,60 @@ def get_stream_info(video_url):
     return stream_info
 
 
+class ChromecastDeviceError(ClickException):
+    pass
+
+
 class Cache:
-    def __init__(self, cache_dir=os.path.join(tempfile.gettempdir(), "catt_cache/")):
+    def __init__(self, duration=3 * 24 * 3600,
+                 cache_dir=os.path.join(tempfile.gettempdir(), "catt_cache")):
         self.cache_dir = cache_dir
         try:
             os.mkdir(cache_dir)
         except:
             pass
 
-    def _get_cache_filename(self, key):
-        return os.path.join(self.cache_dir, key)
+        cache_filename = os.path.join(cache_dir, "chromecast_hosts")
```

Comment:
I was using a function for no particular reason, but since you're directly assigning, might as well just do:

~~~
self.cache_filename = os.path.join(cache_dir, "chromecast_hosts")
~~~

I think that's quite clear and works well.